### ISSUE TITLE
Add support for Default values

### ISSUE THREAD

**Issue by michalonderko** [Kentico/kontent-management-sdk-js#79]
### Motivation

Default values for the Text and Taxonomy element are already added to Kontent.
Default values for Date and Time, Number and Multiple choice element are going to be released by the end of June 22.

### Proposed solution

Mapi documentation:
Text element: https://kontent.ai/learn/reference/management-api-v2/#section/Text-type-element
Taxonomy element: https://kontent.ai/learn/reference/management-api-v2/#section/Taxonomy-type-element

Date and time element default value:
```
"default":{
    "global":{
        "value": string <date-time> Nullable
    }
}
```

Number element default value:
```
"default":{
    "global":{
        "value": number <float> Nullable
    }
}
```

Multiple choice element default value: 
```
"default":{
    "global":{
        "value": Array<string <{ id, codename, external_id }>> // equivalent to taxonomy
    }
}
```

**Comment by michalonderko** [comment]
Please, add code samples also.

**Comment by Enngage** [comment]
Released in `2.8.0`
